compose/ui/ui/src/jvmMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.jvm.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.input.pointer.util
+
+/**
+ * Some platforms (e.g. iOS) velocity calculation is incorrect when end position is used in velocity calculation.
+ */
+internal actual val onlyPressedEventsForVelocityTracker: Boolean
```

Comment:
Why do we need it in `jvmMain` source set? `notMobile` should cover it

Reply:
We don't, indeed, it's a compilation error.

Reply:
But we need not forget to add actual to Android for upstreaming